### PR TITLE
OGM-1541 Upgrade Jackson to 2.9.5 (WF 13.0.0)

### PR DESCRIPTION
**Pull request by fax4ever**
https://hibernate.atlassian.net/browse/OGM-1541

### CODE REVIEW

File: bom/pom.xml [hunk 1]
```diff
@@ -52,7 +52,7 @@
         <version.org.jboss.logging.jboss-logging-tools>2.1.0.Final</version.org.jboss.logging.jboss-logging-tools>
         <version.com.fasterxml.classmate>1.3.3</version.com.fasterxml.classmate>
         <version.org.jboss.resteasy>3.0.24.Final</version.org.jboss.resteasy> <!-- Used by CouchDB and Neo4j remote -->
-        <version.com.fasterxml.jackson>2.8.9</version.com.fasterxml.jackson>
+        <version.com.fasterxml.jackson>2.9.7</version.com.fasterxml.jackson>
```
Comment (gsmet) [inline bom/pom.xml]: The idea here is to be in line with the WF dependencies.

So not sure how we should deal with this.

Comment (fax4ever) [reply bom/pom.xml]: Let me check the one provided by WF.

Comment (fax4ever) [reply bom/pom.xml]: WildFly 13.0.0 uses jackson-core-2.9.5.jar

**Comment by DavideD** [comment]
Merged https://github.com/hibernate/hibernate-ogm/pull/1095

Thanks

**Comment by fax4ever** [comment]
Thanks @DavideD !

**Pull request merged**